frontend/src/views/ReportBugView.vue

```diff
@@ -2,8 +2,10 @@
 import { onMounted } from 'vue';
 import LoadingSpinner from '@/elements/LoadingSpinner.vue';
 
+const redirectUrl = import.meta.env?.VITE_REPORT_BUG_URL ?? '/';
+
 const redirect = async () => {
-  window.location = import.meta.env?.VITE_REPORT_BUG_URL ?? '/';
+  window.location.replace(redirectUrl);
```

Comment:
Switched to .replace so you're less likely to fight with your back button.

Reply:
Convenient, thanks!